docs-src/references/strongly-typed-errors/intro.md

```diff
@@ -0,0 +1,9 @@
+---
+id: intro
+---
+
+This reference lists possible [Workflow Task](/tasks#workflow-task) errors and how to resolve them.
```

Comment:
```suggestion
This reference lists possible [Workflow Task](/tasks/#workflow-task) errors and how to resolve them.
```

Reply:
Actually this should link to `/concepts/what-is-a-workflow-task` and link magic auto rewrites to this.

But `/` before `#` doesn't work any more -